hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/ListingBasedRollbackRequest.java

```diff
@@ -20,10 +20,12 @@
 
 import org.apache.hudi.common.util.Option;
 
+import java.io.Serializable;
+
 /**
  * Request for performing one rollback action.
  */
-public class ListingBasedRollbackRequest {
+public class ListingBasedRollbackRequest implements Serializable {
```

Comment:
Is it possible to add a test case for this change, i.e., a test with Spark engine context to show that without the `Serializable` the test fails, while adding this is necessary to pass the test?